Stripe/ApplePay/STPAPIClient+ApplePay.m

```diff
@@ -25,47 +71,19 @@ + (NSData *)formEncodedDataForPayment:(PKPayment *)payment {
         [[[NSString alloc] initWithData:payment.token.paymentData encoding:NSUTF8StringEncoding] stringByAddingPercentEncodingWithAllowedCharacters:set];
     __block NSString *payloadString = [@"pk_token=" stringByAppendingString:paymentString];
 
-    if (payment.billingAddress) {
-        NSMutableDictionary *params = [NSMutableDictionary dictionary];
-        
-        NSString *firstName = (__bridge_transfer NSString*)ABRecordCopyValue(payment.billingAddress, kABPersonFirstNameProperty);
-        NSString *lastName = (__bridge_transfer NSString*)ABRecordCopyValue(payment.billingAddress, kABPersonLastNameProperty);
-        if (firstName.length && lastName.length) {
-            params[@"name"] = [NSString stringWithFormat:@"%@ %@", firstName, lastName];
-        }
-        
-        ABMultiValueRef addressValues = ABRecordCopyValue(payment.billingAddress, kABPersonAddressProperty);
-        if (addressValues != NULL) {
-            if (ABMultiValueGetCount(addressValues) > 0) {
-                CFDictionaryRef dict = ABMultiValueCopyValueAtIndex(addressValues, 0);
-                NSString *line1 = CFDictionaryGetValue(dict, kABPersonAddressStreetKey);
-                if (line1) {
-                    params[@"address_line1"] = line1;
-                }
-                NSString *city = CFDictionaryGetValue(dict, kABPersonAddressCityKey);
-                if (city) {
-                    params[@"address_city"] = city;
-                }
-                NSString *state = CFDictionaryGetValue(dict, kABPersonAddressStateKey);
-                if (state) {
-                    params[@"address_state"] = state;
-                }
-                NSString *zip = CFDictionaryGetValue(dict, kABPersonAddressZIPKey);
-                if (zip) {
-                    params[@"address_zip"] = zip;
-                }
-                NSString *country = CFDictionaryGetValue(dict, kABPersonAddressCountryKey);
-                if (country) {
-                    params[@"address_country"] = country;
-                }
-                CFRelease(dict);
-                [params enumerateKeysAndObjectsUsingBlock:^(NSString *key, NSString *obj, __unused BOOL *stop) {
-                    NSString *param = [NSString stringWithFormat:@"&card[%@]=%@", key, [obj stringByAddingPercentEncodingWithAllowedCharacters:set]];
-                    payloadString = [payloadString stringByAppendingString:param];
-                }];
-            }
-            CFRelease(addressValues);
-        }
+    if (payment.billingAddress && payment.shippingAddress == nil) {
```

Comment:
@benzguo my only reason for keeping all 3 options was so that if both the `shippingAddress` and the `billingAddress` were set then we would pick the `billingAddress`, doing this keeps with the same behaviour that is seen at present.